### PR TITLE
Update ways of working in the localisation team

### PR DESCRIPTION
**Pull request by BatoolMM**
### Summary

Add more information about the new structure of the calls we have in the localisation team and decision making.

### What should a reviewer concentrate their feedback on?

- [ ] *Lorem ipsum dolor sit amet, consectetur adipiscing.*
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

**Review by malvikasharan — APPROVED** [review]

File: book/website/community-handbook/translation/translation-workflow.md [hunk 1]
```diff
@@ -4,13 +4,40 @@
...
+- **Objectives**:
+  - A facilitator assists with onboarding in the breakout room.
+  - It is an 'Ask Me Anything' call for any questions.
+  - The main room is dedicated to working on milestones for the localisation work.
+  - Tasks are prepared in advance (in Crowdin Kanban) to support new members.
+### Community Meeting
+- **Frequency**: This is integrated into the TTW Collaboration Café. 
+- **Chair**: Selected from leads
+- **Membership**: Open to anyone interested in localisation.
+- **Objectives**: A forum to discuss deliverables/outputs (e.g. toolkits, glossary, blog post, ...etc), consult the community, and seek feedback.
```
Comment (da5nsy) [inline book/website/community-handbook/translation/translation-workflow.md:26]:
```suggestion
- **Objectives**: A forum to discuss deliverables/outputs (for example: toolkits, glossary, blog posts), consult the community, and seek feedback.
```

File: book/website/community-handbook/translation/translation-workflow.md [hunk 2]
```diff
@@ -4,13 +4,40 @@
...
+- **Frequency**: Once every month
+- **Chair**: Selected from leads
+- **Membership**: Open to anyone interested in localisation
+- **Objectives**:
+  - A facilitator assists with onboarding in the breakout room.
+  - It is an 'Ask Me Anything' call for any questions.
+  - The main room is dedicated to working on milestones for the localisation work.
+  - Tasks are prepared in advance (in Crowdin Kanban) to support new members.
+### Community Meeting
+- **Frequency**: This is integrated into the TTW Collaboration Café.
```
Comment (aleesteele) [inline book/website/community-handbook/translation/translation-workflow.md:23]:
```suggestion
- **Frequency**: This is integrated into the _The Turing Way_ Collaboration Café. 
```
Just using the full name, as we're aiming to phase out the shortened name!

File: book/website/community-handbook/translation/translation-workflow.md (hunk 1 above)
Comment (aleesteele) [reply book/website/community-handbook/translation/translation-workflow.md:26]: Nice catch - I've done the same, and it breaks our compiler I believe?

**Review by bsipocz — APPROVED** [review]
Once CI is all green, this can go ahead.

**Comment by bsipocz** [comment]
Thanks @BatoolMM!

**Pull request merged**